test/test_torch.py

```diff
@@ -6047,6 +6047,19 @@ def test_Size(self):
         self.assertIsInstance(x[:-1], torch.Size)
         self.assertIsInstance(x + x, torch.Size)
 
+    def test_Size_scalar(self):
+        three = torch.tensor(3, dtype=torch.int64)
```

Comment:
you don't need to specify the dtypes anymore (they will automatically be inferred to int64).